monty/bot.py

```diff
@@ -10,8 +10,10 @@
 import redis.asyncio
 import sqlalchemy as sa
 from disnake.ext import commands
+from disnake.ext.commands.core import CogT
```

Comment:
This isn't a public attribute of disnake. If we need to use it for type checking, an issue needs to be raised upstream